app/src/main/java/org/simple/clinic/search/PatientSearchScreenController.kt

```diff
@@ -20,15 +27,22 @@ private typealias Ui = PatientSearchScreen
 private typealias UiChange = (Ui) -> Unit
 
 class PatientSearchScreenController @Inject constructor(
-    private val repository: PatientRepository
+    private val repository: PatientRepository,
+    private val dobValidator: DateOfBirthFormatValidator
 ) : ObservableTransformer<UiEvent, UiChange> {
 
   override fun apply(events: Observable<UiEvent>): ObservableSource<UiChange> {
-    val replayedEvents = events.compose(ReportAnalyticsEvents()).replay().refCount()
+    val replayedEvents = events
+        .compose(ReportAnalyticsEvents())
+        .replay()
+        .refCount()
+        .compose(validateQuery())
```

Comment:
We have reporting for all validation errors in `Patient Entry`. Should we be reporting these errors too?

Reply:
This is already being done inside `showValidationErrors()`